### PR TITLE
Translation Dandelion_en for Dandelion_pt-br

### PR DESCRIPTION
**Pull request by nelsonrenan**

### CODE REVIEW

**Comment by lh1008** [comment]
Hey @nelsonrenan thank you for the translation. Pinging @botoserelepe @netrik182 for review.

**Review by botoserelepe — CHANGES_REQUESTED** [review]
@nelsonrenan very nice translation! :) I opted to keep the name "Dandelion" without translation, as it is a software name. Besides that, I proposed only small changes in your text, I hope you agree with them.

File: monero-outreach-docs/translations/pt-br/archive/dandelion_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+﻿# Dandelion++ for Monero
+
+**“Dandelion++ impede que agentes do mal associem seu endereço IP às suas transações com Monero.” - 03/02/2020**  
+
+Monero fica em sua rede de comunicação par a par. A rede de computadores que armazenam a Blockchain – chamados de nós – compartilham informações que suprem a rede Monero, como os endereços de nós, dados históricos da blockchain, e novas transações a serem adicionadas aos blocos. Os nós são identificados pelos endereços IP (Internet Protocol na sigla em inglês), expondo assim ao risco de que observadores possam associar endereços IP’s às transações, retirando o anonimato dos dados que eles contêm. Dente-de-leão++ é um método para ocultar esta conexão que está planejado para o próximo lançamento do software Monero. Este artigo descreve como Dente-de-leão++ funciona e o que irá fazer pelos usuários do Monero.
```
[the remark this answers is not in the document]
Comment (nelsonrenan) [reply monero-outreach-docs/translations/pt-br/archive/dandelion_pt-br.md:6]: Perfect! I had thought about it, but I decided to translate and wait for a suggestion in the review.

**Comment by lh1008** [comment]
hey @nelsonrenan, is everything ready or does it need something else?

**Comment by nelsonrenan** [comment]
Hi @lh1008. Yes, is everything ready.

**Review by netrik182 — APPROVED** [review]
First line still has the original title. Besides that, should be good to go.

**Comment by nelsonrenan** [comment]
Hi @netrik182. The first line will remain as in the original. As recommended by @botoserelepe .

**Review by botoserelepe — APPROVED** [review]

**Review by lh1008 — CHANGES_REQUESTED** [review]
Before merging @nelsonrenan could you please squash commits and make only one. :)

**Comment by erciccione** [comment]
@lh1008 It's ok, i can squash the commits as i usually do for the PRs on this repo. I just need an approval.

**Review by lh1008 — APPROVED** [review]
Approved

**Pull request merged**